src/components/notice/Notice.tsx

```diff
@@ -17,7 +17,7 @@ const AminoNotice = styled.div`
 
   &,
   & > * {
-    font-weight: 500;
+    font-weight: base;
```

Comment:
```suggestion
    font-weight: 500;
```
I think there is no value `base` for the `font-weight` attribute 🤔

Reply:
Hah! Thanks for catching that. It was a find and replace bug. 🐛